### PR TITLE
Add method to get response from Schnorr protocol

### PR DESCRIPTION
**Pull request by lovesh**
Signed-off-by: lovesh <lovesh.bond@gmail.com>

### CODE REVIEW

File: libzmix/src/signatures/bbs/pok_sig.rs [hunk 1]
```diff
@@ -360,4 +377,80 @@ mod tests {
...
+        let same_blinding = FieldElement::random();
+
+        let mut blindings_1 = FieldElementVector::random(message_count - 1);
+        blindings_1.insert(1, same_blinding.clone());
+
+        let mut blindings_2 = FieldElementVector::random(message_count - 1);
+        blindings_2.insert(4, same_blinding.clone());
+
+        // Blinding for the same message is kept same
+        assert_eq!(blindings_1[1], blindings_2[4]);
```
Comment (brentzundel) [inline libzmix/src/signatures/bbs/pok_sig.rs]: Do the blindings need to be the same in order for the Schnorr proof to work?  I thought it was sufficient for the holder to know the blindings for each hidden value used in the equality proof.

Comment (lovesh) [reply libzmix/src/signatures/bbs/pok_sig.rs]: The blindings need to be same when proving equality. The response for each `message` is of the form `blinding + challenge*message` where `challenge` is common across all messages (and statements). The verifier expects the response for same messages to be same.

Comment (brentzundel) [reply libzmix/src/signatures/bbs/pok_sig.rs]: How does that work with link secrets, which are blinded differently in each credential? I feel like I'm missing something obvious.

**Comment by lovesh** [comment]
Link secrets are given the same blinding during proofs.

**Comment by mac-arrap** [comment]
@lovesh can you please resolve merge conflicts and then please feel free to merge

**Pull request merged**